### DISCUSSION TITLE
w3 icon in footer section

### DISCUSSION THREAD

**Discussion by 2div** [tailwindlabs/tailwindcss#11425] [Help] [answered]
Dear Community

Kindly advise how to fix following issue.

https://play.tailwindcss.com/PKaKwMtlax

The home icon under address is became more smaller after adding more text 

How can i keep it size same as other icon below it ?

Thank you

**Comment by wongjn** [comment]
Apply `flex-shrink: 0` via `shrink-0` to the `<svg>`: https://play.tailwindcss.com/wx2QUyGQQk

**Reply by 2div** [reply]
Thank you its work

**Comment by 2div** [comment]
I have another question
if i have grid of 5 cols 
how can i adjust the width of the 2n 3rd 4th cols width smaller so that i can give extra width to the 5th col , i just need little more width for the 5th col ?

Thanks

**Reply by wongjn** [reply, marked as answer]
If you are using a CSS grid layout, you could consider defining the sizing of grid column tracks as you'd like by setting the `grid-template-columns` property on the grid container element.
